rosplan_planning_system/src/STN_tool/replan_loop.py

```diff
@@ -57,11 +60,10 @@
         print "Service call failed: %s"%e
         replans += 1
 
-if goal_achieved:
-    print "SUCCESS ", str(replans)
-    result = open(problem_path[:-5] + '_result_batch' +str(experiment_batch) +'.txt', 'w')
-    result.write('SUCCESS  '+ str(replans) + '\n')
-else:
-    print "FAILED ", str(replans)
-    result = open(problem_path[:-5] + '_result_batch' + str(experiment_batch) +'.txt', 'w')
-    result.write('FAILED  '+ str(replans) + '\n')
+file_path = "~/" + os.path.basename(problem_path)[:-5] + '_result_batch' +str(experiment_batch) +'.txt'
+elapsed = time.time() - snap
+elapsed_sim = rospy.get_time()
+success = "SUCCESS" if goal_achieved else "FAILED"
+with open(file_path, 'wt') as result:
```

Comment:
Here we need to use os.path.expanduser to make tilda work (on my cluster it does not work)
See: https://stackoverflow.com/a/30151425

Reply:
Can you check the latest commit - is it correct now?

Reply:
It should be perfect. I'll try now.